release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -92,6 +92,13 @@ For more information about enabling a feature set during installation, see xref:
 ==== {product-title} on ARM
 {product-title} {product-version} is now supported on ARM architecture-based Azure installer-provisioned infrastructure. AWS Graviton 3 processors are now available for cluster deployments and are also supported on {product-title} 4.11. For more information about instance availability and installation documentation, see xref:../installing/installing-preparing.html#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms]
 
+[id="ocp-4-12-oc-mirror-oci"]
+==== Mirroring Operator images in OCI format with the oc-mirror CLI plug-in (Technology Preview)
+
+Using the oc-mirror CLI plug-in to mirror Operator images in OCI format instead of Docker v2 format is now available as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview].
```

Comment:
```suggestion
Using the `oc-mirror` CLI plug-in to mirror Operator images in OCI format instead of Docker v2 format is now available as a link:https://access.redhat.com/support/offerings/techpreview[Technology Preview].
```

Reply:
In the section that you xref too, it looks like `oc-mirror` is used when refering to the `oc-mirror` CLI plug-in.

Reply:
Actually most of the places we haven't been using backticks, because we refer to it generically too. So I'll leave it not backticked for now, but I might go back and remove the backticks from those few places that do use it, for consistency. Thanks!